examples/gno.land/p/demo/subscription/subscription.gno

```diff
@@ -0,0 +1,123 @@
+// Package subscription provides a library for managing different types of
```

Comment:
I love this, I think we should enforce more of these detailed docs with packages.
Have you considered moving this to `doc.gno`?